ui/ducks/confirm-transaction/confirm-transaction.duck.js

```diff
@@ -308,3 +344,38 @@ export function clearConfirmTransaction() {
     type: CLEAR_CONFIRM_TRANSACTION,
   };
 }
+
+export const updateTxValueIfMaxEthSettled = (
+  dispatch,
+  state,
+  { gasFeeEstimates, gasEstimateType },
+) => {
+  const txId = state.confirmTransaction?.txData?.id;
```

Comment:
I wonder about all the digging deep into state in this function: are there existing selectors to get this data? Should there be?